pkg/apis/api/types.go

```diff
@@ -756,6 +756,8 @@ type RiskLevel struct {
 
 var FailureRiskLevelNone = RiskLevel{Name: "None", Level: 0}
 var FailureRiskLevelLow = RiskLevel{Name: "Low", Level: 1}
-var FailureRiskLevelMedium = RiskLevel{Name: "Medium", Level: 5}
-var FailureRiskLevelUnknown = RiskLevel{Name: "Unknown", Level: 7}
-var FailureRiskLevelHigh = RiskLevel{Name: "High", Level: 10}
+var FailureRiskLevelUnknown = RiskLevel{Name: "Unknown", Level: 25}
+var FailureRiskLevelMedium = RiskLevel{Name: "Medium", Level: 50}
+var FailureRiskLevelIncompleteTests = RiskLevel{Name: "IncompleteTests", Level: 75}
```

Comment:
Should be have a bigger difference between incompleteTests (level: 70)and missingData (level: 80)??

Reply:
My feeling is they were similar in that something was missing.  Overall I wanted to spread the ranges out more but I do think we should cluster the things that are common.  If we had another missing / incomplete we could make it 74 or 77, etc. but I definitely wanted to have room between High and MissingData for things that might come up that we consider a higher risk.